cities/hilden.json

```diff
@@ -56,7 +56,7 @@
   "metadata": {
     "data_source": {
       "title": "Stadt Hilden",
-      "url": "https://hildener-wochenmaerkte.de"
+      "url": "https://www.hilden.de/de/wirtschaft-bauen/citymanagement/wochenmaerkte/"
```

Comment:
If Hilden and Potsdam are stable you can decrease the `ACCEPTABLE_WARNINGS_COUNT` to `1` here:

https://github.com/wo-ist-markt/wo-ist-markt.github.io/blob/master/validation/markets-validator.js#L55

Not sure if Potsdam will be stable. It popped up as an error very often in the last years.

Reply:
Don't you want to change the constant?

Reply:
Changed to 1, comment to mallorca, which fails here on this machine (see issue #670)